build-logic/src/main/kotlin/com.hedera.hashgraph.application.gradle.kts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2016-2022 Hedera Hashgraph, LLC
+ *
+ * This software is the confidential and proprietary information of
+ * Hedera Hashgraph, LLC. ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Hedera Hashgraph.
+ *
+ * HEDERA HASHGRAPH MAKES NO REPRESENTATIONS OR WARRANTIES ABOUT THE SUITABILITY OF
+ * THE SOFTWARE, EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+ * TO THE IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE, OR NON-INFRINGEMENT. HEDERA HASHGRAPH SHALL NOT BE LIABLE FOR
+ * ANY DAMAGES SUFFERED BY LICENSEE AS A RESULT OF USING, MODIFYING OR
+ * DISTRIBUTING THIS SOFTWARE OR ITS DERIVATIVES.
+ */
+
+plugins {
+    id("application")
```

Comment:
Use Gradle's 'application' plugin instead of custom solution to define "main" class.